lua/entities/base_wire_entity.lua

```diff
@@ -324,15 +323,17 @@ end
 -- It also allows us to only update overlays when someone is looking at the entity.
 
 function ENT:SetOverlayText( txt )
-	if not self.OverlayData then
-		self.OverlayData = {}
+	local overlayData = self.OverlayData
+	if not overlayData then
+		overlayData = {}
+		self.OverlayData = overlayData
```

Comment:
Can we just set self.OverlayData in ENT:Initialize and remove this block?

Reply:
This doesn't seem to be quite so simple. Initializing OverlayData elsewhere seems to cause issues clientside with at least E2s (but possibly more entities as well).